src/collections/_documentation/workflow/releases/index.md

```diff
@@ -334,3 +334,74 @@ You can also use our [API]({%- link _documentation/api/releases/post-release-dep
 ## Release Artifacts
 
 JavaScript and iOS projects can utilize release artifacts to unminify or symbolicate error stack traces. To learn more, please check out our [iOS]({%- link _documentation/clients/cocoa/index.md -%}#sentry-cocoa-debug-symbols) and [JavaScript]({%- link _documentation/platforms/javascript/sourcemaps.md -%}) docs.
+
+## Release Health
+
+{% capture __alert_content -%}
+Sentry’s Release Health features are currently in beta. For more details about access to these features, feel free to reach out at [LINK: email?].
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#crash), and [session data]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
```

Comment:
Should we add a section with the highlevel overiew of how does it work? What about this:

How does it work?
Once configured our mobile SDKs will send health data about the usage of your application. 
The basic entity that we work with is a session that represents interaction of the user with the application. 
The session is started when your user opens your application and ends with the end of the application or long inactivity.
The sessions are submitted to sentry so that you can track the usage and adoption of your application.
When a user of your application experiences a crash, error or abnormal exit the session will be flagged accordingly 
and derived metrics like number of the number of the users that did not experience a crash In the specified time range are calculated.
The metrics are always connected to the specific version of your application so that you can match them with the specific version of your code. 
To learn more about the details please check <link>